src/snarky/wrapper.node.js

```diff
@@ -9,12 +9,14 @@ function getWasm() {
   return globalThis.jsoo_runtime.plonk_wasm;
 }
 
+// TODO get rid of shutdown
 let didShutdown = false;
 async function shutdown() {
-  if (global.wasm_rayon_poolbuilder && !didShutdown) {
-    didShutdown = true;
-    global.wasm_rayon_poolbuilder.free();
-    await Promise.all(global.wasm_workers.map((worker) => worker.terminate()));
-    process.exit(0);
-  }
+  process.exit(0);
```

Comment:
`shutdown` as previously implemented failed to always shut down.. so I just made it exit the node process directly